### DISCUSSION TITLE
Where to find the commit of a specific release?

### DISCUSSION THREAD

**Discussion by yakMM** [osmandapp/OsmAnd#15114] [Q&A] [answered]
Hello,

I'm looking for the sepcific commit of a release (including minor version, i.e. 4.2.7).

I can see the branch r4.2 is for the release 4.2, but there is no tag for the minor? Is the branch built at the time the minor relase is prepared and that's all?

Thanks

**Comment by vshcherb** [comment, marked as answer]
It's not 100% perfect for different markets it could be different commits and same release. We do not need mostly that and you can find commit by release date. We assume 4.2 branch is stable 4.2 release and every next commit makes it only more stable :-)

**Reply by yakMM** [reply]
Thanks it's all I need to know. I can then take the branch rx.x, latest commit and have a stable build.
